5calls/app/src/main/java/org/a5calls/android/a5calls/adapter/OutcomeAdapter.java

```diff
@@ -0,0 +1,75 @@
+package org.a5calls.android.a5calls.adapter;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import org.a5calls.android.a5calls.R;
+import org.a5calls.android.a5calls.model.Outcome;
+
+import java.util.List;
+
+public class OutcomeAdapter extends RecyclerView.Adapter<OutcomeAdapter.ViewHolder> {
+
+    private List<Outcome> outcomes;
+    private boolean enabled;
+    private Callback callback;
+
+    public OutcomeAdapter(List<Outcome> outcomes, boolean enabled, Callback callback) {
+        this.outcomes = outcomes;
+        this.enabled = enabled;
+        this.callback = callback;
+    }
+
+    public OutcomeAdapter(List<Outcome> outcomes, Callback callback) {
+        this(outcomes, true, callback);
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
+        notifyDataSetChanged();
+    }
+
+    @Override
+    public OutcomeAdapter.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        return new ViewHolder(LayoutInflater.from(parent.getContext()).inflate(R.layout.item_outcome, parent, false));
```

Comment:
please wrap lines at 100 chars here and below.

Reply:
Done.